components/collective-page/hero/HeroAvatar.js

```diff
@@ -81,14 +81,33 @@ const Translations = defineMessages({
     id: 'collective.settings',
     defaultMessage: 'Settings',
   },
+  uploadImage: {
+    id: 'uploadImage.sizeRejected',
+    defaultMessage: 'Image resolution needs to be less than 3000x3000, and file size must be below 5mo.',
```

Comment:
I think there is a typo in line 86: 5mo/5mb

Reply:
More precisely, it should be 5MB (uppercase), see https://en.wikipedia.org/wiki/Megabyte